### DISCUSSION TITLE
Libraries & CLI Category Name

### DISCUSSION THREAD

**Discussion by shannona** [BlockchainCommons/Community#23] [General]
We've got an increasing number of reference code projects and libraries such as seedtool and keytool that exercise them. We need to be able to group them together under some meaningful category name that implies what they are and draws them together.

So, I brainstormed some possibilities. Additional ideas or thoughts or decisions from @wolfmcnally and @ChristopherA  would be very welcome.

**Possible Titles:** Crypto Keys; Crypto Building Blocks; Crypto Atoms; Nuts & Bolts; Crypto Tacks; Crypto Blocks; Crypto Prime; Crypto Premier; ArchCrypto; Crypto Fundamentals; Crypto Commons; Crypto Library; Crypto Athenaeum.

*or:* Wallet Building Blocks; Wallet Atoms; Wallet Tacks; Wallet Blocks; Wallet Prime; Wallet Premier; ArchWallet; Wallet Fundamentals; Wallet Commons; Wallet Library; Wallet Athenaeum.

*or:* Airgap Building Blocks; Airgap Atoms; Airgap Tacks; Airgap Blocks; Airgap Prime; Airgap Premier; Airgap Fundamentals; Airgap Commons; Airgap Library; Airgap Athenaeum.

*or:* Blockchain Keys; Blockchain Building Blocks; Blockchain Atoms; Blockchain Tacks; Blockchain Blocks; Blockchain Prime; Blockchain Premier; ArchBlockchain; Blockchain Fundamentals; Blockchain Library; Blockchain Athenaeum.

**Possible Subtitles:** This would take the form of "A B & C" from the below list:

*A:* Crypto, Blockchain, Wallet, Airgap
*B:* Libraries, Reference Code, Code
*C:* Samples, CLI Utilities, Utilities, Examples

So, possibilities for subtitles could "Crypto Libraries & Samples" or "Wallet Reference Code & CLI Utilities"

Thoughts? Preferences for title and subtitles? Things I totally missed?

**Comment by wolfmcnally** [comment]
Hm, I thought I discussed with @ChristopherA the idea of bringing these all together under the "Gordian" brand?
